cmd/lakefs/cmd/run.go

```diff
@@ -91,6 +93,13 @@ var runCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		logger := logging.Default()
 		cfg := loadConfig()
+		viper.WatchConfig()
+		viper.OnConfigChange(func(in fsnotify.Event) {
+			lvl := viper.GetString("logging.level")
+			fmt.Println("Changing log level to", lvl)
+			logging.SetLevel(lvl)
```

Comment:
Do we want to handle an empty `lvl` string (in case that `GetString` returned such one)?

Reply:
We handle it here the same way we handle it on startup -> Let viper decide.

Reply:
What do we do if SetLevel failed?  I think we should at least issue an error here.

Reply:
It can't fail and it doesn't return an error.